libs/blocks/merch/merch.css

```diff
@@ -8,6 +8,8 @@ span[data-wcs-osi] {
 }
 
 span.placeholder-resolved[data-template="strikethrough"], span.price.price-strikethrough {
```

Comment:
isn't `span.price.price-strikethrough` enough? or is to keep layout room before price is loaded?

Reply:
@npeltier I did not change this line, and not sure I want to change it now, because I also don't know the reason why .placeholder-resolved is there, maybe it's needed :)

Reply:
ho true, sorry, ok :)